bucket/gsudo.json

```
@@ -0,0 +1,18 @@
+{
+    "version": "0.4.1",
+    "description": "A Sudo for Windows - run elevated without spanning a new Console Host Window",
```

No need for explanation

```suggestion
    "description": "A Sudo for Windows",
```